### PR TITLE
Add PhysicalZTag

### PR DESCRIPTION
**Pull request by dstrain115**
- Simple class in cirq.google.ops that can denote physical Z gates.
- Just includes the class definition so far, no functionality.

### CODE REVIEW

File: cirq/google/ops/physical_z_tag.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A class that can be used to denote a physical Z gate."""
+from cirq import protocols
+
+
+class PhysicalZTag:
+    """Class to add as a tag onto an Operation to denote a Physical Z operation.
+
+  By default, all Z rotations on Google devices are considered to be virtual.
```
Comment (maffoo) [inline cirq/google/ops/physical_z_tag.py]: nit: 4-space indent.

File: cirq/google/ops/physical_z_tag.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  When performing the Z operation, the device will update its internal phase
+  tracking mechanisms, essentially commuting it forwards through the circuit
+  until it hits a non-commuting operation (Such as a sqrt(iSwap)).
+
+  By attaching this to a Z rotation operation, this class will denote to the
```
Comment (maffoo) [inline cirq/google/ops/physical_z_tag.py]: nit: "When applied to a Z rotation operation, this tag indicates to the hardware that..."

Comment (dstrain115) [reply cirq/google/ops/physical_z_tag.py]: Done.

File: cirq/google/ops/physical_z_tag.py (hunk 1 above)
Comment (dstrain115) [reply cirq/google/ops/physical_z_tag.py]: Done.

**Pull request merged**